docs/tutorials/FlowTutorial.md

```diff
@@ -55,6 +55,7 @@ following commands:
 
 ```
 cd OpenROAD-flow-scripts
+git clean -xdi
```

Comment:
This is a dangerous command (as previously noted).  I do not think we should suggest users use this unless they really need to and understand the implications.